CpEpEngine.cpp

```diff
@@ -1283,7 +1283,7 @@ STDMETHODIMP CpEpEngine::GetFingerprints(TextMessage* msg, SAFEARRAY** keylist)
 
             fprCombined = keyid + colon + fpr;
             free(_keylist->value);
-            _keylist->value = strdup(fprCombined.c_str());
+            _keylist->value = _strdup(fprCombined.c_str());
```

Comment:
What is happening here? `strdup` is part of the core wrapper API, and since you are dealing with core struct, it looks like the function to use.

Reply:
I guess that this change is for ensuring better compatibility with the Microsoft VC++ environment and avoids potential warnings or issues related to POSIX compliance, right?
On Windows, using _strdup is recommended to avoid compatibility issues or compiler warnings while in a Unix-like system (Linux, macOS), strdup is the standard and typically preferred.
So afaik when working in a Windows environment using Visual Studio, the Microsoft-specific _strdup, is safer .